docker-compose.yaml

```diff
@@ -6,7 +6,7 @@ services:
       - 4001:4001
       - 8080:8080
       - 5001:5001
-  vitalam-node:
+  dscp-node:
     image: ghcr.io/digicatapult/vitalam-node:v2.11.0
```

Comment:
need to wait until `vitalam-node` is renamed and built until we change or this repo's test workflow fails

same applies to `helm-lint` and `helm-test` in workflows which point to the `node` e.g. `ct lint --validate-maintainers false --charts helm/${{ needs.repo_ids.outputs.repo_name }}/ --chart-repos vitalam-node=https://digicatapult.github.io/vitalam-node/`